spec/features/user_searches_with_80202_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe "User" do
+  it "searches with 80202" do
+    visit root_path
+
+    fill_in "search", with: '80202'
+    click_on "Find Events"
+
+    expect(current_path).to eq(search_path)
+
+    expect(page).to have_content('16 events for March, sorted by popularity')
+
+    expect(page).to have_content("Event Name")
```

Comment:
all the content that you're testing for is static and on the page no matter what. what could we add to make this test more robust?